rplugin/python3/chadtree/transitions.py

```diff
@@ -693,6 +693,21 @@ def p_pre() -> Dict[str, str]:
 
         pre_existing = await run_in_executor(p_pre)
         if pre_existing:
+            # If path(s) already exist, allow user to rename the destination
+            for source, dest in pre_existing.items():
+                def ask_rename() -> bool:
+                    resp = nvim.funcs.input("Path exist!!! Rename: ", dest)
+                    return resp
+                new_dest = await call(nvim, ask_rename)
+                if new_dest:
+                    operations[source] = new_dest
+                else:
+                    return None
```

Comment:
@[ms-jpq](https://github.com/ms-jpq) wouldn't it be nice to ask wether it's ok to replace the existing files instead?